scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -587,10 +592,50 @@ <h6>No pinned variants</h6>
         </li>
       {% endfor %}
     </ul>
-    </div>
+    {{ clinvar_vars_summary(suspects, case, institute) }}
+      </div>
   </div>
 {% endmacro %}
 
+{% macro clinvar_var_status(variant, case, institute) %}
+    {% if variant._id and variant.category != 'str' %}
+        <form id="clinvar_submit" class="d-flex justify-content-center" action="{{ url_for('clinvar.clinvar_add_variant', institute_id=institute._id, case_name=case.display_name) }}" method="POST">
+            {% if case.clinvar_variants and variant._id in case.clinvar_variants.keys() %}
+                (included in ClinVar submission)
+            {% else %}
+                <button type="submit" name="var_id" value="{{variant._id}}" class="btn btn-secondary btn-sm" style="float: right;">Add to ClinVar submission</button>
+            {% endif %}
+        </form>
+    {% endif %}
+{% endmacro %}
+
+{% macro clinvar_vars_summary(suspects, case, institute) %}
+    {% if case.clinvar_variants %}
+        {% set suspects_ids, submitted_ids = [], [] %}
+        {% for entry in suspects %}
+            {% do suspects_ids.append(entry.get("_id")) %}
+        {% endfor %}
+        {% for entry in case.clinvar_submitted_variants if entry.get("_id") not in suspects_ids %}
+            {% do submitted_ids.append(entry.get("_id")) %}
+        {% endfor %}
+        {% if submitted_ids | length >0 %}
+            <div class="d-flex align-content-center flex-wrap">
+                <div class="col-auto d-flex align-items-center wrap">
+                    <h6 class="m-2">&nbsp;Variants submitted to ClinVar, not present in the table above:</h6>
+                </div>
+                {% for entry in case.clinvar_submitted_variants if entry.get("_id") not in suspects_ids %}
+                    <div class="m-2">{{ pretty_link_variant(entry, case) }}</div>
+                {% endfor %}
+            </div>
```

Comment:
I feel there must be a more efficient way to do this but have had some trouble with reaching the intended result in other ways. The intended result being that the header is rendered once if there are variants submitted to ClinVar which is not already listed in the table and that it is followed by links to each of the relevant variants.